Tests/Microsoft.AppCenter.Test.Windows/Storage/MockStorage.cs

```diff
@@ -47,6 +47,7 @@ public Task DeleteLogs(string channelName)
         {
             lock (this)
             {
+                AppCenterLog.Debug(AppCenterLog.LogTag, $"Storage.DeleteLogs for channelName={channelName}");
```

Comment:
Is this actually printed in test logs? Just curious.

Reply:
yes